client/webpack.config.node.js

```diff
@@ -135,7 +135,6 @@ const webpackConfig = {
 		} ),
 		new webpack.NormalModuleReplacementPlugin( /^lib[/\\]abtest$/, 'lodash/noop' ), // Depends on BOM
```

Comment:
We may want to remove the `lib/abtest` and `lib/analytics` aliases as well in future PRs.